UnityProject/Assets/Scripts/Messages/Server/TileChangesNewClientSync.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+//long name I know. This is for syncing new clients when they join to all of the tile changes
+public class TileChangesNewClientSync : ServerMessage
+{
+	public static short MessageType = (short) MessageTypes.TileChangesNewClientSync;
+	public string data;
+	public NetworkInstanceId ManagerSubject;//fixme: Recipient is redundant! Can be safely removed
```

Comment:
hehe, why use ManagerSubject if it's redundant?

Reply:
I struggled with that name for ages. Its actually the TileChangeManager in question

Reply:
As you can eventually have more then 1 station which can have construction / deconstruction on it

Reply:
Ah, I guess that fixme is not valid here then

Reply:
oh right I didn't even see that. I cut and pasted one of your messages and altered it